### PR TITLE
Update Object.values browser versions

### PR DESCRIPTION
**Pull request by radeno**
Based on
- https://caniuse.com/#search=object.values
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values

### CODE REVIEW

File: polyfills/Object/values/config.json [hunk 1]
```diff
@@ -3,21 +3,21 @@
 		"es2017"
 	],
 	"browsers": {
-		"edge": "<15",
-		"edge_mob": "<15",
-		"ie": "9 - 14",
+		"edge": "< 12",
```
Comment (JakeChampion) [inline polyfills/Object/values/config.json]: This should be `<14` according to MDN.
image

```suggestion
		"edge": "< 14",
```

File: polyfills/Object/values/config.json [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
+		"edge_mob": "< 12",
```
Comment (JakeChampion) [inline polyfills/Object/values/config.json]:
```suggestion
		"edge_mob": "< 14",
```

File: polyfills/Object/values/config.json [hunk 3]
```diff
@@ -3,21 +3,21 @@
...
+		"ie": "*",
 		"ie_mob": "*",
 		"chrome": "< 54",
-		"ios_chr": "*",
-		"safari": "*",
-		"ios_saf": "*",
+		"ios_chr": "< 54",
+		"safari": "< 10.1",
+		"ios_saf": "< 10.1",
```
Comment (JakeChampion) [inline polyfills/Object/values/config.json]: This should be `< 10.3` according to MDN
image
```suggestion
		"ios_saf": "< 10.3",
```

**Comment by radeno** [comment]
@JakeChampion You are right. CanIUse  shows bad versions. I have pushed PR to their repo.
Official info https://docs.microsoft.com/en-us/microsoft-edge/dev-guide/whats-new/edgehtml-14
Edge mobile embed system rendering engine. So it is questionable if version 14 supports it or not.

**Pull request merged**